### PR TITLE
Update Thrift bindings and remove ununsed imports

### LINKED ISSUE #116: Remove TFrozenDict from Thrift generated files
`TFrozenDict` not used within pymapd for anything, but the import of it within the mapd module can cause errors (especially when trying to use pymapd through the R reticulate package).

While these files are auto-generated, it feels like we can safely remove `TFrozenDict` and potentially other imports, in order to optimize/minimize loading issues.

### PR DESCRIPTION
**Pull request by randyzwitch**
Fixes #116 by removing imports that are aren't being used from the thrift files. This is a minor fix, as it really becomes an issue when trying to use thrift via R via the Python bindings.

Will leave open until #141 can be evaluated with a new Docker build (i.e. this PR not expected to break things, but better to wait than assume nothing else fails)

### CODE REVIEW

**Comment by andrewseidl** [comment]
Do you have a script to do this automatically? Otherwise you might be back in the same place the next time the thrift bindings get updated (which it's probably time to do that again anyways).

File: pymapd/connection.py [hunk 1]
```diff
@@ -399,7 +398,7 @@ def create_table(self, table_name, data, preserve_index=False):
 
         row_desc = build_row_desc(data, preserve_index=preserve_index)
         self._client.create_table(self._session, table_name, row_desc,
-                                  TTableType.DELIMITED)
+                                  TTableType.DELIMITED, TCreateParams())
```
Comment (randyzwitch) [inline pymapd/connection.py]: @andrewseidl When I updated the Thrift bindings, `create_table` now has an additional kwarg for `create_params`. Is this okay to pass this inside the `create_table` python method as `TCreateParams()` (which defaults to `NULL` for the `is_replicated` argument)  without making it a user-specified function? Doing it this way passes the test, so technically it works.

Or to put it another way, is this something a user of pymapd should have the ability to customize? If so, I can add an argument and update the docstring.

Comment (andrewseidl) [reply pymapd/connection.py]: Yeah, defaulting it to null should be fine, though `False` would be better.

It's only relevant when running distributed with Enterprise Edition.

Comment (randyzwitch) [reply pymapd/connection.py]: Ok, I'll make it `False` but not have it as a keyword argument in the function.

**Pull request merged**